rts/lra/lra-test/lra-archive-provider/src/main/java/io/narayana/lra/archive/LraArchiveProvider.java

```diff
@@ -0,0 +1,37 @@
+package io.narayana.lra.archive;
+
+import io.narayana.lra.client.internal.proxy.nonjaxrs.LRACDIExtension;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+
+public class LraArchiveProvider {
+
+    public static Archive<?> createLraArchive() {
+        JavaArchive archive = ShrinkWrap.create(JavaArchive.class);
+
+        // adding LRA spec interfaces under the Thorntail deployment
+        archive
+            .addPackages(true, org.eclipse.microprofile.lra.annotation.Compensate.class.getPackage());
+
+        // adding Narayana LRA implementation under the WildFly Swarm deployment
```

Comment:
```suggestion
        // adding Narayana LRA implementation under the Thorntail deployment
```

Reply:
I believe this was done so resolving the conversation